src/main/resources/static/assets/js/sdh.js

```diff
@@ -0,0 +1,68 @@
+(function() {
+	var app = new Vue({
+		el: '#contents',
+		data: {
+			fields: [{key:'domain', sortable:true}, {key:'date_last_assessed', sortable:true}, {key:'clinician_priority', sortable:true}, {key:'patient_readiness', sortable:true}],
+			filters: {
+			      domain: '',
+			      date_last_assessed: '',
+			      clinician_priority: '',
+			      patient_readiness: ''
+			    },
+	        items: [
+	            { domain: 'Food Security', date_last_assessed: '01/01/2018', clinician_priority: 'High', patient_readiness: 'Not Ready' },
+	            { domain: 'Transportation', date_last_assessed: '01/01/2019', clinician_priority: 'Low', patient_readiness: 'Motivated'},
+	            { domain: 'Housing Instability', date_last_assessed: '10/11/2018', clinician_priority: 'Medium', patient_readiness: 'Ready' },
+	            { domain: 'Utilities', date_last_assessed: '02/01/2019', clinician_priority: 'High', patient_readiness: 'Ready' },
+	            { domain: 'Interpersonal Violence', date_last_assessed: '12/01/2018', clinician_priority: 'Medium', patient_readiness: 'Motivated' },
+	            { domain: 'Education', date_last_assessed: '04/04/2018', clinician_priority: 'Medium', patient_readiness: 'Not Ready' },
+	            { domain: 'Financial Strain', date_last_assessed: '03/17/2012', clinician_priority: 'Low', patient_readiness: 'Ready' }
+	          ]
+		},
+		methods: {
+			_priorityVariant (priority) {
+				if (priority == 'High') {
```

Comment:
These should probably be `===` comparisons. Also, the `_priorityVariant` and `_readinessVariant` methods will both return `undefined` if no case matches. Will that be a problem?

Reply:
Thanks, I'll do some testing but probably best to return the empty string for clarity.